middle_end/flambda2/terms/zero_alloc_attribute.ml

```diff
@@ -35,21 +35,13 @@ let print ppf t =
     Format.fprintf ppf "@[assert_zero_alloc%s@]"
       (if strict then "_strict" else "")
 
-let from_lambda : Lambda.zero_alloc_attribute -> Location.t -> t =
- fun a loc ->
+let from_lambda : Lambda.zero_alloc_attribute -> t =
```

Comment:
Can we now have `type t = Lambda.zero_alloc_attribute` and make this the identity function rather than building new copies?

Reply:
ah, nice! done.